godtools/App/Features/ToolSettings/Presentation/ToolSettings/Views/ToolSettingsOptions/ToolSettingsOptionsViewModel.swift

```diff
@@ -5,8 +5,37 @@
 //  Created by Levi Eggert on 5/11/22.
 //
 
-import Foundation
+import SwiftUI
 
-class ToolSettingsOptionsViewModel {
+class ToolSettingsOptionsViewModel: BaseToolSettingsOptionsViewModel {
     
+    private let trainingTipsEnabled: Bool
+    
+    private weak var flowDelegate: FlowDelegate?
+    
+    required init(flowDelegate: FlowDelegate, trainingTipsEnabled: Bool) {
+        
+        self.flowDelegate = flowDelegate
+        self.trainingTipsEnabled = trainingTipsEnabled
+        
+        super.init()
+        
+        trainingTipsTitle = trainingTipsEnabled ? "Hide tips" : "Training tips"
```

Comment:
I'm assuming these will be translated later?  As far as process, do you typically build out a feature and then have a separate ticket to go back and address translations?

Reply:
Hey @rachaelblue there hasn't been a specific process for translating.  I am planning to translate those as the final step in this task here -> https://jira.cru.org/browse/GT-1568.